WTWSMS/common/nicknames/VIET_immersion_nicknames.txt

```diff
@@ -158,7 +164,12 @@ nick_the_realized = {
 
 nick_the_deluded = {
 	allow = {
-		religion = buddhist
+		OR = {
+			religion = buddhist
+			religion = priscillianism
+			religion = donatist
+			religion = monatism
```

Comment:
Because it was a part of a Buddhist flavour expansion and thus meant to be for them.